build-support/docker/nicer-shell-environment.sh

```diff
@@ -0,0 +1,32 @@
+# This script is intended to be sourced, not executed!
```

Comment:
Why not bake this into the image?

Reply:
Well, you are kinda enforcing it, no? Having the file be in the image doesn't mean that people have to use it when running a container.